src/main/java/core/basesyntax/model/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.model;
+
+import core.basesyntax.model.abstraction.Color;
+
+public class IsoscelesTrapezoid extends Rectangle {
```

Comment:
```suggestion
public class IsoscelesTrapezoid {
```
IsoscelesTrapezoid **IS NOT** a Rectangle